metadata/service/exporter/configurable/exporter.go

```diff
@@ -61,12 +62,19 @@ func (exporter *MetadataServiceExporter) Export(url *common.URL) error {
 		version, _ := exporter.metadataService.Version()
 		exporter.lock.Lock()
 		defer exporter.lock.Unlock()
+
+		protocolConfig, ok := config.GetRootConfig().Protocols[constant.DefaultProtocol]
+		if !ok {
+			protocolConfig = config.NewProtocolConfigBuilder().
+				SetName(constant.DefaultProtocol).
+				SetPort(strconv.Itoa(config.GetApplicationConfig().MetadataServicePort)).
+				Build()
+		}
```

Comment:
If the user sets the MetadataServicePort, should we use that port？

Reply:
consistent with java

Reply:
Adding a default port to the current policy causes CI failures, need delete default port?

Reply:
default port cause ci fail ,link: https://github.com/apache/dubbo-go/actions/runs/7191807715/job/19588269066

Reply:
since we want reuse dubbo protocol port, I think the priority should be
1. MetadataServicePort
2. dubbo protocol port
3. default dubbo protocol port

Reply:
use default port triple protocol for fail

Reply:
use default port,ci fail

Reply:
https://github.com/apache/dubbo-go/actions/runs/7203720804/job/19624025841

Reply:
please create a new branch to show how you use default port,I want to know why ci failed

Reply:
https://github.com/apache/dubbo-go/commit/d08bf0f07367d307ae3a8e9f5b49885643b94757